### ISSUE TITLE
FPN not working

### ISSUE THREAD

**Issue by mletombe** [qubvel/segmentation_models#66]
Hi, 

With the exact same inputs and backbone, this code is working for Unet but not for FPN :

model = Unet(backbone_name=backbone, classes=1, encoder_weights='imagenet')
#model = FPN(backbone_name=backbone, classes=1, encoder_weights='imagenet')
model.compile('Adam', 'binary_crossentropy', ['binary_accuracy'])
model.fit(x, y, epochs=10, batch_size=1)

**For FPN :**
Epoch 1/10
100/100 [==============================] - 21s 213ms/step - loss: 15.6301 - binary_accuracy: 0.0196
Epoch 2/10
100/100 [==============================] - 14s 143ms/step - loss: 15.6301 - binary_accuracy: 0.0196
Epoch 3/10
100/100 [==============================] - 14s 143ms/step - loss: 15.6301 - binary_accuracy: 0.0196
Epoch 4/10
100/100 [==============================] - 14s 143ms/step - loss: 15.6301 - binary_accuracy: 0.0196
Epoch 5/10
100/100 [==============================] - 14s 143ms/step - loss: 15.6301 - binary_accuracy: 0.0196
Epoch 6/10
100/100 [==============================] - 14s 144ms/step - loss: 15.6301 - binary_accuracy: 0.0196
Epoch 7/10
100/100 [==============================] - 14s 144ms/step - loss: 15.6301 - binary_accuracy: 0.0196
Epoch 8/10
100/100 [==============================] - 14s 144ms/step - loss: 15.6301 - binary_accuracy: 0.0196
Epoch 9/10
100/100 [==============================] - 14s 144ms/step - loss: 15.6301 - binary_accuracy: 0.0196
Epoch 10/10
100/100 [==============================] - 14s 144ms/step - loss: 15.6301 - binary_accuracy: 0.0196

**For Unet:**
Epoch 1/10
100/100 [==============================] - 21s 214ms/step - loss: 0.1810 - binary_accuracy: 0.9673
Epoch 2/10
100/100 [==============================] - 14s 135ms/step - loss: 0.0266 - binary_accuracy: 0.9955
Epoch 3/10
100/100 [==============================] - 14s 135ms/step - loss: 0.0172 - binary_accuracy: 0.9957
Epoch 4/10
100/100 [==============================] - 14s 136ms/step - loss: 0.0141 - binary_accuracy: 0.9958
Epoch 5/10
100/100 [==============================] - 14s 136ms/step - loss: 0.0136 - binary_accuracy: 0.9956
Epoch 6/10
100/100 [==============================] - 14s 136ms/step - loss: 0.0165 - binary_accuracy: 0.9942
Epoch 7/10
100/100 [==============================] - 14s 136ms/step - loss: 0.0123 - binary_accuracy: 0.9959
Epoch 8/10
100/100 [==============================] - 14s 136ms/step - loss: 0.0090 - binary_accuracy: 0.9970
Epoch 9/10
100/100 [==============================] - 14s 136ms/step - loss: 0.0081 - binary_accuracy: 0.9973
Epoch 10/10
100/100 [==============================] - 14s 136ms/step - loss: 0.0075 - binary_accuracy: 0.9975

Is there anything broken?

Thanks for this repo!

**Comment by qubvel** [comment]
Hi @mletombe
For one class set `activation='sigmoid'`
Let me know if that helps.

**Comment by mletombe** [comment]
Yes it works.

Thanks a lot!
